MovingBlocks/Terasology
# Feature: Albanian translation

Hello,

### Contains

this PR contains the Albanian translation of the game.

- menu_sq.lang : was added to the engine ressources assets folder 

- flag_sq.png : ALbanian flag added to the assets flags folder

### How to test

- Open the game

- Go to the Settings

- Choose Albanian in the langages menu list

- play the game to see translation

Thank you.

Aizen93 and fortiseven
